### PR TITLE
CCR Careers Smoke test: Fill In An Application As An Internal User , Verify CCR Landing Page items

### PR DESCRIPTION
**Pull request by alenaminsk**

### CODE REVIEW

File: src/test/java/CustomBusiness/CCR/Pages/CCRAdminUserPage.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+        /**  field Position Title**/
+        @FindBy(xpath = "//input[@id='title']")
+        public WebElement fieldPositionTitle;
+
+        /**  field Description**/
+        @FindBy(xpath = "//body[@data-id='description']//p")
+        public WebElement fieldDescription;
+
+        /**  Todays Date Actiev Datepicker Choice**/
```
Comment (hamidnesar) [inline src/test/java/CustomBusiness/CCR/Pages/CCRAdminUserPage.java:31]: Please replace Actiev with Active

File: src/test/java/CustomBusiness/CCR/Pages/CCRApplicantPage.java [hunk 2]
```diff
@@ -168,7 +172,39 @@ public CCRApplicationPage() {
     @FindBy(xpath = "//select[@id='business_country']")
     public WebElement countryBusinessDropdown;
 
-    /** Degree Dropdown **/
+    /** Applicant Degree Degree Dropdown **/
     @FindBy(xpath = "//select[@id='degree']")
     public WebElement degreeDropdown;
+
+    /** Reference Degree Degree Dropdown **/
```
Comment (hamidnesar) [inline src/test/java/CustomBusiness/CCR/Pages/CCRApplicantPage.java:179]: Repeated word - Degree

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, 
Please review the comments

File: src/test/java/CustomBusiness/CCR/Pages/CCRAdminUserPage.java (hunk 1 above)
Comment (alenaminsk) [reply src/test/java/CustomBusiness/CCR/Pages/CCRAdminUserPage.java:31]: fixed

File: src/test/java/CustomBusiness/CCR/Pages/CCRApplicantPage.java (hunk 2 above)
Comment (alenaminsk) [reply src/test/java/CustomBusiness/CCR/Pages/CCRApplicantPage.java:179]: fixed

**Review by hamidnesar — APPROVED** [review]
Reviewed and approved

**Review by sarwarahmed1 — APPROVED** [review]

**Review by lfcabrera14 — APPROVED** [review]
reviewed and approved

**Pull request merged**